docker-compose.yml

```diff
@@ -1,97 +1,32 @@
-version: '2.4'
-
 services:
-  nginx-proxy:
-    image: jwilder/nginx-proxy
-    ports:
-      - "80:80"
-    volumes:
-      - /var/run/docker.sock:/tmp/docker.sock:ro
```

Comment:
Since the application is no longer running on a Docker container, we can no longer bind the Nginx proxy to the application web server, so removing this service.